Makefile

```diff
@@ -1,11 +1,16 @@
 
 QUAY_USERNAME ?= 
 QUAY_PASSWORD ?= 
-POSTGRESQL_OPERATOR_VERSION ?= 0.0.3
-POSTGRESQL_OPERATOR_IMAGE ?= quay.io/$(QUAY_USERNAME)/postgresql-operator
+
+ifdef OPERATOR_TESTING
+POSTGRESQL_APPR_NAMESPACE ?= $(QUAY_USERNAME)-testing
```

Comment:
As a general suggestion, I think the word `POSTGRESQL_` in front of every variable is not needed, you could very well remove it.

Reply:
Good point... removed
Thanks.